x/auth/stdtx.go

```diff
@@ -28,11 +29,35 @@ func NewStdTx(msgs []sdk.Msg, fee StdFee, sigs []StdSignature, memo string) StdT
 	}
 }
 
-//nolint
+// GetMsgs returns the all the transaction's messages.
 func (tx StdTx) GetMsgs() []sdk.Msg { return tx.Msgs }
 
+// ValidateBasic does a simple and lightweight validation check that doesn't
+// require access to any other information.
+func (tx StdTx) ValidateBasic() sdk.Error {
```

Comment:
Should we do the number-of-signatures check here too?

Reply:
Agreed :+1:

Reply:
Ahhh yes! Good catch!